src/main/java/internetshop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,146 @@
+package internetshop.dao.jdbc;
+
+import internetshop.controller.LoginController;
+import internetshop.dao.OrderDao;
+import internetshop.exception.DataProcessingException;
+import internetshop.lib.Dao;
+import internetshop.model.Order;
+import internetshop.model.Product;
+import internetshop.util.conection.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.apache.log4j.Logger;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    private static final Logger LOGGER = Logger.getLogger(LoginController.class);
+
+    @Override
+    public Order create(Order order) {
+        String query = "INSERT INTO internetshop.orders (user_id) VALUES (?)";
+        String query2 =
+                "INSERT INTO internetshop.orders_products (order_id, product_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement =
+                    connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            preparedStatement.setLong(1, order.getUserId());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                order.setId(resultSet.getLong(1));
+            }
+            PreparedStatement preparedStatement2 = connection.prepareStatement(query2);
+            for (Product product: order.getProducts()) {
+                preparedStatement2.setLong(1, order.getId());
+                preparedStatement2.setLong(2, product.getId());
+                preparedStatement.executeUpdate();
+            }
+            LOGGER.info("Order has been created");
+        } catch (SQLException e) {
+            throw new DataProcessingException("Order creation has failed", e);
+        }
+        return order;
+    }
+
+    @Override
+    public Optional<Order> get(Long id) {
+        String query =
+                "SELECT internetshop.orders.*, internetshop.orders_products.product_id, "
+                + "internetshop.products.name, internetshop.products.price "
+                        + "FROM internetshop.orders "
+                + "INNER JOIN internetshop.orders_products ON orders.id = orders_products.order_id "
+                + "INNER JOIN internetshop.products ON orders_products.product_id = products.id "
```

Comment:
let's remove database name from queries everywhere, we are setting it while establishing connection